src/endpoints/account-memberships.js

```diff
@@ -10,6 +11,27 @@ class AccountMembershipsEndpoint extends BaseExtend {
     )
   }
 
+  All(accountId, token = null, headers = {}) {
+    const { includes, sort, limit, offset, filter } = this
```

Comment:
You are using filter and other props here, but I don't see how a user can ever set them? Can you add some tests to show how filtering is supposed to be used.